samples/bluetooth/central_iso/src/main.c

```diff
@@ -0,0 +1,244 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <errno.h>
+#include <zephyr.h>
+#include <sys/printk.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/uuid.h>
+#include <bluetooth/gatt.h>
+#include <bluetooth/iso.h>
+#include <sys/byteorder.h>
+
+#define MAX_ISO_APP_DATA (CONFIG_BT_ISO_TX_MTU - BT_ISO_CHAN_SEND_RESERVE)
+
+static void start_scan(void);
+
+static struct bt_conn *default_conn;
+static struct k_delayed_work iso_send_work;
+static struct bt_iso_chan iso_chan;
+NET_BUF_POOL_FIXED_DEFINE(tx_pool, 1, CONFIG_BT_ISO_TX_MTU, NULL);
+
+/**
+ * @brief Send ISO data on timeout
+ *
+ * This will send an increasing amount of ISO data, starting from 1 octet.
+ *
+ * First iteration : 0x00
+ * Second iteration: 0x00 0x01
+ * Third iteration : 0x00 0x01 0x02
+ *
+ * And so on, until it wraps around the configured ISO TX MTU (CONFIG_BT_ISO_TX_MTU)
+ *
+ * @param work Pointer to the work structure
+ */
+static void iso_timer_timeout(struct k_work *work)
+{
+	int ret;
```

Comment:
There's a good reason for that:
I used `err` if the value return is only success with value = 0 (typical).  
I used `ret` for `bt_iso_chan_send` as that returns the number of bytes sent, so that is a return value and not just an error value.